cli/src/query.rs

```diff
@@ -42,41 +46,52 @@ pub fn query_files_at_paths(
                 query_cursor.captures(&query, tree.root_node(), text_callback)
             {
                 let capture = mat.captures[capture_index];
+                let capture_name = &query.capture_names()[capture.index as usize];
                 writeln!(
                     &mut stdout,
                     "    pattern: {}, capture: {}, row: {}, text: {:?}",
                     mat.pattern_index,
-                    &query.capture_names()[capture.index as usize],
+                    capture_name,
                     capture.node.start_position().row,
                     capture.node.utf8_text(&source_code).unwrap_or("")
                 )?;
+                results.push(query_testing::CaptureInfo {
+                    name: capture_name.to_string(),
+                    position: capture.node.start_position(),
+                });
```

Comment:
Right now, the `tree-sitter query` command prints out all of the match info, formatted as plain text. Do you think we want to see this output when running tests? I'm wondering if query *testing* should be a different code path, that just runs the assertions, instead of printing all of the debug output.

Reply:
I think it’s helpful to see all the captures next to the failures. Don’t have strong feelings, though.